sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/login/WebLoginTOTPConfigurationTest.java

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.commons.model.login;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import com.mercedesbenz.sechub.commons.model.JSONConverter;
+
+class WebLoginTOTPConfigurationTest {
+
+    @Test
+    void default_values_are_used_correctly_during_json_serialization_and_deserialization() {
+        /* prepare */
+        WebLoginTOTPConfiguration expectedConfig = new WebLoginTOTPConfiguration();
+
+        /* execute */
+        String json = JSONConverter.get().toJSON(expectedConfig);
+        WebLoginTOTPConfiguration config = JSONConverter.get().fromJSON(WebLoginTOTPConfiguration.class, json);
+
+        /* test */
+        assertEquals(config.getSeed(), expectedConfig.getSeed());
+        assertEquals(config.getValidityInSeconds(), expectedConfig.getValidityInSeconds());
+        assertEquals(config.getTokenLength(), expectedConfig.getTokenLength());
+        assertEquals(config.getHashAlgorithm(), expectedConfig.getHashAlgorithm());
+    }
```

Comment:
Hm... what if "default" values would be null here... would not exist
please make the default constants public (as mentioned before) and check like this:

```java 
assertEquals(config.getSeed(), null); // there is no default
assertEquals(config.getTokenLength(), WebLoginTOTPConfiguration.DEFAULT_TOKEN_LENGTH); 
...
```